### ISSUE TITLE
Some channels unable to load with local API

### ISSUE THREAD

**Issue by LynHyper** [FreeTubeApp/FreeTube#1534]
Some channels seem to be completely unable to load with the local API, when it fallbacks to invidious the channel will have no videos, searching for the channel's videos in the search bar and watching them works completely fine however.

Vsauce3 use to load, albeit after falling back to invidious, and would show videos on the channel completely fine. I think the YouTube Premium-only videos might be the cause of videos not being loaded for the channel.

**To Reproduce:**
1. Go to Vsauce3's channel
2. The channel will fallback to invidious saying "This channel does not exist"

**Link to video:**
https://invidious.snopyta.org/channel/UCwmFOfFuvRPI112vR5DNnrA

**Environment Information:**
 - OS: Arch Linux
 - FreeTube version: 0.13.2 beta
 - Installation Method: FlatHub
 - Primary API Used: Local API

**Error log:** (Sorry if this information is vague, not sure how to gather the error logs)
Local API Error: TypeError: Cannot read property 'simpleText' of undefined
Local API Error: [object Object]
Invidious API Error: This channel does not exist

**Comment by LynHyper** [comment]
Just tried viewing the channel today, apparently it's still falling back to invidious, but, the videos are showing again. The behaviour where the channel "doesn't exist" seems to be inconsistent, it also seems to go against my original guess that it's because of YouTube Premium exclusives.

**Comment by Imaskiller** [comment]
> Just tried viewing the channel today, apparently it's still falling back to invidious, but, the videos are showing again. The behaviour where the channel "doesn't exist" seems to be inconsistent, it also seems to go against my original guess that it's because of YouTube Premium exclusives since videos are showing despite them still being there.

I could reproduce the error for this channel. A specific youtube-premium video causes this issue for me. I made a simple pull request that fixes this issue for me but definitely needs more insight.

**Comment by GilgusMaximus** [comment]
Yes, there seems to be a problem

**Comment by LynHyper** [comment]
Should I close this issue given the pull request exists or should I leave this open until the pull request is merged with mainline?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Just leave this open until the PR is merged.

**Comment by GilgusMaximus** [comment]
Yes please. Especially due to the huge number of pr's being open at the moment, we will get duplicates if the issues are closed prematurely

**Comment by Winterhuman** [comment]
@jasonhenriquez Might need to reopen this issue. The channel `UCfYRNVPQLq-8dGPf5qIWU8w` (channel name: Discordian) isn't loading for FreeTube v0.16.0 Beta, it errors with:

```
TypeError: Cannot read properties of undefined (reading 'response')
```

It's a different error compared to what this issue observed though, maybe this needs to be a separate issue, please say if it should be.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Already fixed in the latest nightly build

**Comment by Winterhuman** [comment]
Oh nice, I didn't think there could of been any patches around this problem since this issue was opened, thanks!
